logiclearner/urls.py

```diff
@@ -9,14 +10,15 @@
 
 
 urlpatterns = [
-    path('', views.IndexView.as_view()),
+    url('^contact/', include('contactus.urls')),
```

Comment:
It looks like the `url` function is deprecated - I would switch this to `path`. I think this should work?

    path('contact/', include('contactus.urls'))

https://docs.djangoproject.com/en/2.2/ref/urls/#url

https://docs.djangoproject.com/en/2.2/ref/urls/#path

Reply:
Oh yea! Got it thank you!